src/NumbersToLetters.java

```diff
@@ -40,9 +40,6 @@ public static void main(String[] args) {
             } else {
```

Comment:
mooi dat je een beveiliging hebt gemaakt voor als users  andere input geven dan x of v. nog beter dat je deze gelijk omzet tot lowercase. Niet vereist voor de opdracht maar je zou nog kunnen overwegen om een beveiliging in te bouwen voor als mensen een ander nummer invoeren dan 0 tot en met  9. nu krijg je als input daarvoor NULL terug.